### PR TITLE
Use the new runtime

### PR DESCRIPTION
**Pull request by vpalatin**
Move the tasklets configuration to the C sources

### CODE REVIEW

File: checksum/dpu/build_configuration.script [hunk 1]
```diff
@@ -1,16 +1,5 @@
...
-
-tasklet add 0..15 medium task_main 2
-
-// Post the input file size into the system mailbox (One 32-bits word), accessible to any thread
-
-sysmbox 1
-
 // DO NOT EDIT BELOW THIS POINT
 
 save $DIRECTORY/$NAME.json
```
Comment (JordiChauzi) [inline checksum/dpu/build_configuration.script]: We should be able to delete this file completely.

File: checksum/dpu/Makefile [hunk 2]
```diff
@@ -49,28 +40,16 @@ directories:
...
-$(EXEC): $(BUILDDIR)/$(BOOTSTRAP).$(OBJEXT) $(C_OBJECTS) $(ASM_OBJECTS)
+$(EXEC): $(C_OBJECTS) $(ASM_OBJECTS)
 	$(CC) -o $@ $^ $(LIB) $(DBG)
 
-#Assemble bootstrap
-$(BUILDDIR)/$(BOOTSTRAP).$(OBJEXT): $(BUILDDIR)/$(BOOTSTRAP).$(ASMEXT)
-	$(CC) -c -o $@ $(CFLAGS) $^ $(INC)
-
 #Compile
 $(ASM_OBJECTS): $(BUILDDIR)/%.$(OBJEXT) : $(SRCDIR)/%.$(ASMEXT)
```
Comment (JordiChauzi) [inline checksum/dpu/Makefile:47]: Nitpick: the assembler sources are handled exactly the same than C sources: can we try to unify the make goals ?

Comment (JordiChauzi) [reply checksum/dpu/Makefile:47]: Maybe more important (but still nothing to do with this PR), I don't see any assembler source file in this project. Do we need this rule at all ?

File: checksum/dpu/build_configuration.script (hunk 1 above)
Comment (vpalatin) [reply checksum/dpu/build_configuration.script]: Yes, sure. fixed.

File: checksum/dpu/Makefile (hunk 2 above)
Comment (vpalatin) [reply checksum/dpu/Makefile:47]: Probably not, maybe it can be seen as an example,
but deleted ...

**Review by JordiChauzi — APPROVED** [review]

**Pull request merged**